Chore/Chore.js

```diff
@@ -105,27 +105,55 @@ function runChoreIntroduction() {
         sendVirtualAssistantMessage('Okay, I will remove this from the list then...')
     }
 
+	if (isVar("ToyGirlfriend") && getVar("ToyGirlfriend"))
+		{
+	sendVirtualAssistantMessage("  I noticed that you have a girlfriend or Wife....");
+	sendVirtualAssistantMessage("  will %girlfriendname% be assigning you chores?");
+
+	answer=createInput("Yes", "no");
+		while (true) {
+			if (answer.isLike("yes")) {
+				 sendVirtualAssistantMessage(" Wonderful");
+				 setVar("AssignedChores", true);
+				break;
+			} else if (answer.isLike("no")) {
+				sendVirtualAssistantMessage("  Too bad, I was hoping "+getVar("GirlfriendName")+ " could play a small part in your submission.");
```

Comment:
Never used. Global variable VARIABLE_SUB_PARTNER_NAME
Furthermore partners aren't yet implemented

Reply:
ummm  legacy code (that works for me) basically you changed variable names,and then replaced code without reimplementing this function. I spent alot of time re-merging code updates from a month or two...  there may be some straggling variable references  (there may be more if we port more of the legacy code)

Reply:
I think there maybe new (old) code in the startup merge that collects girlfriend name